src/type-utils.ts

```diff
@@ -16,6 +16,10 @@ export interface SbvrType<Read = unknown, Write = any, DbWrite = unknown> {
 	fetchProcessing?: FetchProcessing<Read>;
 	validate: Validate<Write, DbWrite>;
 }
+export interface TsTypes<Read, Write> {
```

Comment:
Would it make sense to allow for DbWriteType as well?

Reply:
I don't expect that to be needed by anything outside of this module which is why I'm not exposing it, but if a use case/need comes up for it then I'd be happy to add it